Task10_Auth/src/main/java/ResponseWrapper.java

```diff
@@ -1,22 +1,24 @@
-import java.util.List;
+import reseponsesDTO.UsersDto;
+
+import java.util.ArrayList;
 
 public class ResponseWrapper {
     private int responseCode;
-    private List<String> responseBody;
+    private ArrayList responseBody;
```

Comment:
I still don't understand, why you're not using Object here

Reply:
resolved